src/parser.rs

```diff
@@ -25,6 +25,9 @@ pub trait Parser<'a, T: Input<'a>> {
     /// Reset a `Parser`.
     fn reset(&mut self);
 
+    /// Returns the queue of all matched `(Token, value)`s
```

Comment:
I usually used full stop at the end of documentatiom descriptions mostly because you might need to use multiple sentences and form paragraphs, so this should have one as well.